setup.py

```diff
@@ -55,7 +55,7 @@
         "tqdm>=4.64.0",
         "ijson>=3.1.4",
         "jsonstreams>=0.6.0",
-        "semver>=2.13.0",
+        "semver>=2.13.0,<3.0.0",
```

Comment:
reason for this change?

Reply:
There were mypy errors with v3.0.0 of semver not explicitly exporting compare and VersionInfo